spec/kintsugi_apply_change_to_project_spec.rb

```diff
@@ -377,6 +410,39 @@
       base_project.save
     end
 
+    it "moves file that is referenced by a target from main group to another group" do
+      file_reference = base_project.main_group.new_reference("bar")
+      base_project.targets[0].source_build_phase.add_file_reference(file_reference)
+      base_project.save
+
+      theirs_project = create_copy_of_project(base_project.path, "theirs")
+      new_group = theirs_project.main_group.find_subpath("new_group", true)
+      file_reference = theirs_project.main_group.find_file_by_path("bar")
```

Comment:
The group can be created in `base_project`, then to find the file you can just do: `theirs_project["bar"]`

Reply:
keeping as discussed